db/migrate/20180627060150_add_column_reply_to_content_in_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnReplyToContentInComment < ActiveRecord::Migration[5.1]
+  def change
+  	add_column :comments, :extra_content, :text
```

Comment:
Could we just keep it in the regular comment body, but use a filter to remove it upon sending a notification email, and use JavaScript to hide it in the HTML views? Just wondering if we can solve this without a db change and the maintenance that may come with?

Reply:
I was thinking, it will become complex if we try to fit both the content into one attribute..of course we can use filter but i was thinking we both the forms will have some maintenance but i am happy doing either ways....what do you think?

Reply:
@jywarren any thoughts?

Reply:
I guess I think we might use the `render` methods to offer a "full" or "trimmed" display for use in different places, and filter on display rather than filter on save. The filter could be like:

```ruby
comment.filter_body(:gmail)
# vs
comment.body
```

maybe?